prow/gerrit/client/client_test.go

```diff
@@ -48,7 +52,99 @@ func (f *fgc) ListChangeComments(id string) (*map[string][]gerrit.CommentInfo, *
 	}
 
 	return &comments, nil, nil
+}
+
+func TestApplyGlobalConfigOnce(t *testing.T) {
+	// Empty opener so *syntTime won't panic.
```

Comment:
s/synt/sync

But also, this comment should probably be just above line 61 where the `opener` variable is actually bound.

Reply:
done